app/src/main/java/uk/co/neuralcubes/neuralates/controller/RobotController.java

```diff
@@ -32,7 +34,16 @@ public void unlink(){
     @Subscribe
     public synchronized void updateAcelerometer(MuseHandler.AccelerometerReading reading){
```

Comment:
Even if they're asynchronous, 'synchronized' wouldn't be needed if they always come from the same thread. Happy to the leave them in if we're not sure though.